appinventor/components/src/com/google/appinventor/components/runtime/Texting.java

```diff
@@ -518,7 +522,7 @@ public boolean GoogleVoiceEnabled() {
               @IntentFilterElement(actionElements = {
                   @ActionElement(name = "com.google.android.apps.googlevoice.SMS_RECEIVED")
               })
-          })
+          }, exported = "false")
```

Comment:
I don't believe this should be changed because in theory the Google Voice app is making this broadcast so this broadcast receiver needs to be exported.

Reply:
Reverted